### ISSUE TITLE
Change Backend into ActiveAdmin and Chart Library

### ISSUE THREAD

**Issue by scicco** [RailsApps/rails-prelaunch-signup#24]
Hi Daniel,

first of all i want to say thank you for your efforts in developing nice tutorials and working examples! 
I've forked your app to integrate activeadmin and HighCharts library, a js one which doesn't use flash.
I need to add real data feed on the chart (now is filled with fake values). 
If you want to take a look you can find it here:

https://github.com/scicco/rails-prelaunch-signup/tree/activeadmin

Maybe you can merge those changes into your app.

Hope to be useful!

Bye

Guido

**Comment by DanielKehoe** [comment]
I looked at HighCharts and wanted to use it but I recall the licensing terms were restrictive (a fee is required for commercial use). Is that still the case?

**Comment by scicco** [comment]
Yeah you're right, but in case of a startup with just one developer you can choose the developer license and pay only 360$. More infos are on this page: http://shop.highsoft.com/highcharts.html
Anyway, is possible to integrate the google chart that you used, replacing highchart, and still use activeadmin.
If it will be useful for you i can try doing it in another branch of my fork.

**Comment by DanielKehoe** [comment]
Yes, HighCharts is out. I would like to find another charting library that doesn't use Flash.

I'll take a closer look at your activeadmin integration. I think it's a good idea.

**Comment by scicco** [comment]
Hi, Daniel! Take a look at Morris.js: http://www.oesmith.co.uk/morris.js/ for the charts. It's Javascript with jquery and raphael as requirements, with BSD license.
